spec/factories.rb

```diff
@@ -3,7 +3,7 @@
         first_name { Faker::Name.first_name }
         last_name { Faker::Name.last_name }
         email { Faker::Internet.email }
-        password { Faker::Crypto.md5 }
+        password {BCrypt::Password.create(User::TEST_PASSWORD)}
```

Comment:
Encrypting the password outside of the model means that anyone who uses the model must also remember to encrypt the password. I am worried that if I were using the model I might forget to do that. One of the cleanest ways I've seen to encrypt the password is to use one of the [activerecord callbacks](https://guides.rubyonrails.org/active_record_callbacks.html). To encrypt the password prior to it being saved in the database.